Tribler/Test/test_remote_torrent_handler.py

```diff
@@ -28,6 +28,8 @@ def setUpPreSession(self):
         super(TestRemoteTorrentHandler, self).setUpPreSession()
         self.config.set_dispersy(True)
         self.config.set_torrent_store(True)
+        self.config.set_megacache(True)
+        self.config.set_torrent_collecting(True)
```

Comment:
@NielsZeilemaker Good point. I think I saw an exception because of the missing RemoteTorrentHandler, but I forgot where. I will remove those two lines.